cumulusci/core/github.py

```diff
@@ -45,7 +45,7 @@ def get_github_api_for_repo(keychain, owner, repo):
     if APP_ID and APP_KEY:
         installation = INSTALLATIONS.get((owner, repo))
         if installation is None:
-            gh.login_as_app(APP_KEY, APP_ID)
+            gh.login_as_app(APP_KEY, APP_ID, expire_in=120)
```

Comment:
Does this change go here?

Reply:
this came from a merge.

Reply:
I think github was confused. This was a change I already made on master and no longer shows up in this diff now that I hit the "update branch" button.